### DISCUSSION TITLE
Using Translatable fields in Nova actions

### DISCUSSION THREAD

**Discussion by takdw** [spatie/nova-translatable#61] [Q&A]
When trying to add Translatable fields into my Nova action `fields` function, I keep getting the error:
```
Call to undefined method Spatie\\NovaTranslatable\\Translatable::resolveForAction()
```

Does this mean I can't add Translatable fields to Nova action fields?

**Comment by kukukk** [comment]
Hi,

I also have this issue. Is there a solution for this?
